app/app.py

```diff
@@ -46,6 +46,11 @@ def _get_configured_parser():
                        default=False, action='store_true',
                        help='repeat the classification on the latest '
                        'user-defined set of drawings')
+
+    group2 = parser.add_mutually_exclusive_group()
```

Comment:
We might change `-s` to `-b/--bitness` and `-sym` to `-s` because `-sym`. Or change `-sym` to `-m`.